tests/qweb/qweb.test.ts

```diff
@@ -207,6 +207,13 @@ describe("t-esc", () => {
     const result = renderToString(qweb, "test", { state: { list: [1, 2] } });
     expect(result).toBe("<span>1,2</span>");
   });
+
+  test("t-esc inside t-call, with t-set outside", () => {
+    qweb.addTemplate("main", `<div><t t-set="v">Hi</t><t t-call="sub"/></div>`);
+    qweb.addTemplate("sub", `<span t-esc="v"/>`);
+    const result = renderToString(qweb, "main");
```

Comment:
I am not sure this does test that the HTML is escaped ?
There is another test:
```js  
test("t-esc is escaped", () => {
    qweb.addTemplate("test", `<div><t t-set="var"><p>escaped</p></t><t t-esc="var"/></div>`);
    const domRendered = renderToDOM(qweb, "test");
    expect(domRendered.textContent).toBe("<p>escaped</p>");
  });
```

Reply:
Not sure it's relevant here as the value is hardcoded anyway (in this kind of scenario, you will either define html content, and use t-raw, or define string content and use t-set). I guess there are already a bunch of tests asserting that t-esc properly escapes the value, even inside a t-call, no?

Reply:
that would be my assumption too, yes, I was just making sure...